src/controller/order-controller.ts

```diff
@@ -41,11 +46,15 @@ async function createOrderHandler(
       {
         amount: {
           currency_code: "USD",
-          value: totalAmount,
+          value: (totalAmount + 5).toString(),
```

Comment:
Instead of hard-coding 5 here. Let's pull this default value from the new shipping-cost.json file. Ex:
```js
import shippingCost from "../data/shipping-cost.json";

// this can be the way to get the value of 5
Number(shippingCost.DEFAULT)
```